timestamp,category,user_id,function,line_number,flag,message,exception
2024-06-10 17:44:03,ERROR,asif24,create_or_pass_folder,199,0,Function create_or_pass_folder error,"Function create_or_pass_folder error
Traceback (most recent call last):
  File ""C:\Users\Asif\Desktop\New_version_CogniLink\app.py"", line 192, in create_or_pass_folder
    container_client.get_blob_client(container_name, user_folder).upload_blob("""")
  File ""C:\Users\Asif\Desktop\New_version_CogniLink\venv\lib\site-packages\azure\core\tracing\decorator.py"", line 78, in wrapper_use_tracer
    return func(*args, **kwargs)
  File ""C:\Users\Asif\Desktop\New_version_CogniLink\venv\lib\site-packages\azure\storage\blob\_blob_client.py"", line 765, in upload_blob
    return upload_block_blob(**options)
  File ""C:\Users\Asif\Desktop\New_version_CogniLink\venv\lib\site-packages\azure\storage\blob\_upload_helpers.py"", line 195, in upload_block_blob
    process_storage_error(error)
  File ""C:\Users\Asif\Desktop\New_version_CogniLink\venv\lib\site-packages\azure\storage\blob\_shared\response_handlers.py"", line 184, in process_storage_error
    exec(""raise error from None"")   # pylint: disable=exec-used # nosec
  File ""<string>"", line 1, in <module>
azure.core.exceptions.HttpResponseError: Value for one of the query parameters specified in the request URI is invalid.
RequestId:a4c59649-a01e-0052-122f-bb68f4000000
Time:2024-06-10T12:14:03.1131691Z
ErrorCode:InvalidQueryParameterValue
queryparametername:snapshot
queryparametervalue:asif24
Content: <?xml version=""1.0"" encoding=""utf-8""?>
<Error><Code>InvalidQueryParameterValue</Code><Message>Value for one of the query parameters specified in the request URI is invalid.
RequestId:a4c59649-a01e-0052-122f-bb68f4000000
Time:2024-06-10T12:14:03.1131691Z</Message><QueryParameterName>snapshot</QueryParameterName><QueryParameterValue>asif24</QueryParameterValue></Error>"
2024-06-10 17:44:03,ERROR,asif24,create_or_pass_folder,208,0,"Function create_or_pass_folder Error creating folder'asif24': Value for one of the query parameters specified in the request URI is invalid.
RequestId:a4c59649-a01e-0052-122f-bb68f4000000
Time:2024-06-10T12:14:03.1131691Z
ErrorCode:InvalidQueryParameterValue
queryparametername:snapshot
queryparametervalue:asif24
Content: <?xml version=""1.0"" encoding=""utf-8""?>
<Error><Code>InvalidQueryParameterValue</Code><Message>Value for one of the query parameters specified in the request URI is invalid.
RequestId:a4c59649-a01e-0052-122f-bb68f4000000
Time:2024-06-10T12:14:03.1131691Z</Message><QueryParameterName>snapshot</QueryParameterName><QueryParameterValue>asif24</QueryParameterValue></Error> ","Function create_or_pass_folder Error creating folder'asif24': Value for one of the query parameters specified in the request URI is invalid.
RequestId:a4c59649-a01e-0052-122f-bb68f4000000
Time:2024-06-10T12:14:03.1131691Z
ErrorCode:InvalidQueryParameterValue
queryparametername:snapshot
queryparametervalue:asif24
Content: <?xml version=""1.0"" encoding=""utf-8""?>
<Error><Code>InvalidQueryParameterValue</Code><Message>Value for one of the query parameters specified in the request URI is invalid.
RequestId:a4c59649-a01e-0052-122f-bb68f4000000
Time:2024-06-10T12:14:03.1131691Z</Message><QueryParameterName>snapshot</QueryParameterName><QueryParameterValue>asif24</QueryParameterValue></Error> 
Traceback (most recent call last):
  File ""C:\Users\Asif\Desktop\New_version_CogniLink\app.py"", line 192, in create_or_pass_folder
    container_client.get_blob_client(container_name, user_folder).upload_blob("""")
  File ""C:\Users\Asif\Desktop\New_version_CogniLink\venv\lib\site-packages\azure\core\tracing\decorator.py"", line 78, in wrapper_use_tracer
    return func(*args, **kwargs)
  File ""C:\Users\Asif\Desktop\New_version_CogniLink\venv\lib\site-packages\azure\storage\blob\_blob_client.py"", line 765, in upload_blob
    return upload_block_blob(**options)
  File ""C:\Users\Asif\Desktop\New_version_CogniLink\venv\lib\site-packages\azure\storage\blob\_upload_helpers.py"", line 195, in upload_block_blob
    process_storage_error(error)
  File ""C:\Users\Asif\Desktop\New_version_CogniLink\venv\lib\site-packages\azure\storage\blob\_shared\response_handlers.py"", line 184, in process_storage_error
    exec(""raise error from None"")   # pylint: disable=exec-used # nosec
  File ""<string>"", line 1, in <module>
azure.core.exceptions.HttpResponseError: Value for one of the query parameters specified in the request URI is invalid.
RequestId:a4c59649-a01e-0052-122f-bb68f4000000
Time:2024-06-10T12:14:03.1131691Z
ErrorCode:InvalidQueryParameterValue
queryparametername:snapshot
queryparametervalue:asif24
Content: <?xml version=""1.0"" encoding=""utf-8""?>
<Error><Code>InvalidQueryParameterValue</Code><Message>Value for one of the query parameters specified in the request URI is invalid.
RequestId:a4c59649-a01e-0052-122f-bb68f4000000
Time:2024-06-10T12:14:03.1131691Z</Message><QueryParameterName>snapshot</QueryParameterName><QueryParameterValue>asif24</QueryParameterValue></Error>"
2024-06-10 17:44:03,INFO,asif24,home,1414,1,User asif24 logged in successfully,
2024-06-10 17:44:03,INFO,asif24,update_bar_chart_from_blob,319,1,Function update_bar_chart_from_blob successfully Return blob_list,
2024-06-10 17:44:07,INFO,asif24,update_bar_chart_from_blob,319,1,Function update_bar_chart_from_blob successfully Return blob_list,
2024-06-10 17:44:08,INFO,asif24,get_data_source,2098,1,table_update route successfully send data,
2024-06-10 17:44:25,INFO,asif24,perform_lda___Q_A,1040,1,QnA topics generated.,
2024-06-10 17:44:25,INFO,asif24,handle_ask_question,1998,1,Answer Generated in 9.101506471633911 seconds,
